docs/api/app.md

```diff
@@ -759,6 +759,21 @@ bar, and on macOS, you can visit it from dock menu.
 
 Clears the recent documents list.
 
+### `app.getRecentDocuments()` _macOS_ _Windows_
```

Comment:
Some APIs use naming like `getAllX`. Maybe something to consider here. I'm fine with either though.